src/main.jsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
-import ReactDOM from "react-dom";
+import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App.jsx";
 
-ReactDOM.render(
+ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
```

Comment:
In line 10, it's not necessary to pass a second argument to the render method since you've already done that when creating the root. You can see this warning message in the console as well.